### PR TITLE
Enable tests in build and get rid of delays

### PR DESCRIPTION
**Pull request by kjarosh**

### CODE REVIEW

File: src/test/kotlin/com/github/davenury/ucac/api/SinglePeersetIntegrationTest.kt [hunk 1]
```diff
@@ -157,14 +151,22 @@ class SinglePeersetIntegrationTest {
...
-                throw RuntimeException()
+                throw RuntimeException("Stop")
             }
             val firstLeaderCallbacks: Map<TestAddon, suspend (ProtocolTestInformation) -> Unit> = mapOf(
                 TestAddon.BeforeSendingAgree to firstLeaderAction
             )
+            val afterHandlingApply: suspend (ProtocolTestInformation) -> Unit = {
+                phaser.arriveAndAwaitAdvance()
+            }
+            val peer2Callbacks: Map<TestAddon, suspend (ProtocolTestInformation) -> Unit> = mapOf(
```
Comment (Rados13) [inline src/test/kotlin/com/github/davenury/ucac/api/SinglePeersetIntegrationTest.kt:162]: We have defined typealias for `suspend (protocolTestInformation: ProtocolTestInformation) -> Unit`

File: src/test/kotlin/com/github/davenury/ucac/api/SinglePeersetIntegrationTest.kt [hunk 2]
```diff
@@ -235,10 +234,17 @@ class SinglePeersetIntegrationTest {
             TestAddon.BeforeSendingApply to firstLeaderAction
         )
 
+        val peer2Action: suspend (ProtocolTestInformation) -> Unit = {
+            phaser.arriveAndAwaitAdvance()
+        }
```
Comment (Rados13) [inline src/test/kotlin/com/github/davenury/ucac/api/SinglePeersetIntegrationTest.kt:239]: Is this name correct, as we refer to it in peer3Callbacks?

**Review by Rados13 — APPROVED** [review]

File: src/test/kotlin/com/github/davenury/ucac/api/SinglePeersetIntegrationTest.kt (hunk 1 above)
Comment (kjarosh) [reply src/test/kotlin/com/github/davenury/ucac/api/SinglePeersetIntegrationTest.kt]: Replaced all references to this type in tests

File: src/test/kotlin/com/github/davenury/ucac/api/SinglePeersetIntegrationTest.kt (hunk 2 above)
Comment (kjarosh) [reply src/test/kotlin/com/github/davenury/ucac/api/SinglePeersetIntegrationTest.kt]: Nope, it was a typo, now it should be correct

**Review by Rados13 — APPROVED** [review]
One test fails. This is a test with delays so maybe we could skip tests that are using delays until we rewrite them this way that they will not use delays?

**Comment by kjarosh** [comment]
Yup, I've disabled tests with delays

**Review by Rados13 — APPROVED** [review]

**Pull request merged**